src/platform/apollolake/include/platform/memory.h

```diff
@@ -358,7 +358,7 @@
 #define PLATFORM_HEAP_SYSTEM		2 /* one per core */
 #define PLATFORM_HEAP_SYSTEM_RUNTIME	2 /* one per core */
 #define PLATFORM_HEAP_RUNTIME		1
-#define PLATFORM_HEAP_BUFFER		3
+#define PLATFORM_HEAP_BUFFER		2 /* APL  has no lp memory enabled yet*/
```

Comment:
This looks like a separate PR ?

Reply:
It is in one PR because turning on debug memory free without this memory fixes results in catastrophic system crash. And i do mean catastrophic since we would try to write to non active memory banks.

Overall it is as a whole a fix for #550

If You insist I can separate theme , but i wouldn't recommend it.